src/modules/video_player/index.js

```diff
@@ -58,6 +70,14 @@ function handlePlayerClick() {
     }, 250);
 }
 
+function handleVisibilityChange() {
+    if (document[hidden]) {
+        $('.player-button--volume:has(.mute-button)').click();
```

Comment:
this feature should not override a user-selected setting. if a user manually mutes and alt-tabs, and alt-tabs back.. this would unmute the player.. which is bad.